lua/compe_ultisnips/init.lua

```diff
@@ -0,0 +1,53 @@
+local compe = require'compe'
+
+local M = {}
+
+function M:get_metadata()
+  return {
+    priority = 50,
+    dup = 0,
+    menu = '[Ultisnips]',
+  }
+end
+
+function M:datermine(context)
+  return compe.helper.datermine(context)
+end
+
+function M:complete(args)
+  if vim.fn.exists("*UltiSnips#SnippetsInCurrentScope") == 0 then
+    args.abort()
+    return
+  end
```

Comment:
It should be moved to datermine or just removed.

Reply:
Once checked if ultisnips is installed or not while registering I don't think there would be any use for it. So, I would remove it.